.rubocop.yml

```diff
@@ -61,6 +61,11 @@ Style/CollectionMethods:
 Style/WordArray:
   Enabled: false
 
+# in rspec, expect{...} is conventional
```

Comment:
Contrived example:
```ruby
expect{object.method_a}.not_to raise_error
expect(object.method_b).to eq 5
```
vs.
```ruby
expect { object.method_a }.not_to raise_error
expect(object.method_b).to eq 5
```
I prefer the alignment.  Making a "word" of `}.not_to` feels off to me (and adding a space is worse).